projects/whats_cooking_good_looking/whats_cooking_good_looking/train_ner_workflow.py

```diff
@@ -0,0 +1,261 @@
+                        [
+                            {
+                            "result": [
+                                {
+                                    "value": {"start": 10, "end": 17, "text": "Chennai", "labels": ["LOC"]},
+                                    "from_name": "label",
+                                    "to_name": "text",
+                                    "type": "labels",
+                                    "origin": "manual",
+                                }
+                            ],
+                            "predictions": [
+                                    {
+                                    "result": {"start": 10, "end": 17, "text": "Chennai", "labels": ["LOC"]},
+                                    "model_version": "dummy",
+                                    }
+                                ],
+                            }
+                        ]
+
+    Returns:
+        dict: mapping {model_name: accuracy}
+
+    """
+    model_acc = dict()
+    model_hits = defaultdict(int)
+    for ls_task in json.loads(labelstudio_tasks):
+        annotation_result = ls_task["result"][0]["value"]
+        for key in annotation_result:
+            if key == "id":
+                annotation_result.pop("id")
+        for prediction in ls_task["predictions"]:
+            model_version = prediction["model_version"]
+            model_hits[model_version] += int(prediction["result"] == annotation_result)
+
+    num_task = len(labelstudio_tasks)
+    for model_name, num_hits in model_hits.items():
+        acc = num_hits / num_task
+        model_acc[model_name] = acc
+        print(f"Accuracy for {model_name}: {acc:.2f}%")
+    return model_acc
+
+
+@task
+def load_tasks(bucket_name: str, source_blob_name: str) -> bytes:
+    """Loads Label Studio annotations.
+
+    Args:
+        bucket_name (str): GCS bucket name where tasks are stored.
+        source_blob_name (str): GCS blob name where tasks are stored.
+
+    Returns:
+        str: json dumped tasks
+    """
+    labelstudio_tasks = download_bytes_from_gcs(
+        bucket_name=bucket_name, source_blob_name=source_blob_name
+    )
+    return labelstudio_tasks
+
+
+@task
+def format_tasks_for_train(labelstudio_tasks: bytes) -> str:
+    """Format Label Studio output to be trained in spacy custom model.
+
+    Args:
+        labelstudio_tasks (str): json dumped labelstudio_tasks
+
+    Returns:
+        str: json dumped train data formatted
+    """
+    train_data = []
+    for ls_task in json.loads(labelstudio_tasks):
+        entities = [
+            (ent["value"]["start"], ent["value"]["end"], label)
+            for ent in ls_task["result"]
+            for label in ent["value"]["labels"]
+        ]
+        if entities != []:
+            train_data.append((ls_task["task"]["data"]["text"], {"entities": entities}))
+    return json.dumps(train_data)
+
+
+@task
+def load_model(
```

Comment:
Since this task is present in apply_ner_workflow.py, you can either import it `from whats_cooking_good_looking.apply_ner_workflow import load_model` or use the [reference task](https://docs.flyte.org/projects/cookbook/en/latest/auto/core/flyte_basics/reference_task.html#sphx-glr-auto-core-flyte-basics-reference-task-py).